tests/conformance/test_conformance.py

```diff
@@ -423,6 +467,131 @@ def blob_create_resumable_upload_session(client, _preconditions, **resources):
         blob.create_resumable_upload_session()
 
 
+def blob_make_private(client, _preconditions, **resources):
+    if _preconditions:
+        pytest.skip(_OBJECT_ACL_PATCH_MSG)
+    bucket = resources.get("bucket")
+    object = resources.get("object")
+    blob = client.bucket(bucket.name).blob(object.name)
+    blob.make_private()
+
+
+def blob_make_public(client, _preconditions, **resources):
+    if _preconditions:
+        pytest.skip(_OBJECT_ACL_PATCH_MSG)
+    bucket = resources.get("bucket")
+    object = resources.get("object")
+    blob = client.bucket(bucket.name).blob(object.name)
+    blob.make_public()
+
+
+def bucket_make_private(client, _preconditions, **resources):
+    if _preconditions:
+        pytest.skip(_BUCKET_ACL_PATCH_MSG)
+    bucket = client.bucket(resources.get("bucket").name)
+    bucket.make_private()
+
+
+def bucket_make_public(client, _preconditions, **resources):
+    if _preconditions:
+        pytest.skip(_BUCKET_ACL_PATCH_MSG)
```

Comment:
These should be never-idempotent so this message should never come up.

Reply:
Filed #582 for adding preconditions and retry config support.